src/app/about/page.tsx

```diff
@@ -0,0 +1,64 @@
+"use client";
+
+import "./page.scss";
+import dynamic from "next/dynamic";
+import { usePageContent } from "app/shared/hooks/usePageContent";
+import { PathOptions } from "app/types/PageContentEnums";
+import { Breadcrumb } from "app/components/shared/Breadcrumb/Breadcrumb";
+import { ROUTE_PATHS } from "app/shared/const";
+
+const SafDivider = dynamic(
+    () =>
+        import("@saffron/core-components/react").then(module => module.SafDivider),
+    { ssr: false }
+);
+
+const SafBreadcrumbItem = dynamic(
+    () =>
+        import("@saffron/core-components/react").then(
+        module => module.SafBreadcrumbItem
+        ),
+    { ssr: false }
+);
+
+export default function About() {
+    const  { content }  = usePageContent(PathOptions.ABOUT);
+
+    if (!content) {
+        return null;
+    }
+    
+    const { title, htmlTopContent, htmlHighlightContent, htmlBodyContent, htmlFooterContent } = content;
+
+    return (
+        <>
+            {/* <HelmetTags title={title} /> */}
```

Comment:
Could you remove this and try to add the Nextjs metadata dinamically ?. You can check an example on src/app/apps/[app]/page.tsx